lava-slave/scripts/start.sh

```diff
@@ -22,4 +18,10 @@ do
 done
 
 # start an http file server for boot/transfer_overlay support
-(cd /var/lib/lava/dispatcher; python -m SimpleHTTPServer 80)
+(cd /var/lib/lava/dispatcher; python -m SimpleHTTPServer 80) &
+
+# FIXME lava-slave does not run if old pid is present
+rm -f /var/run/lava-slave.pid
+service lava-slave start || exit 5
+
+sleep 3650d
```

Comment:
What is this sleep for?

Reply:
Without it, docker will exit.

Reply:
could we keep sleep for the moment and rework after 2018.x since lots of process will be run differently due to systemd transition ?

Reply:
Yes, that sounds good.